### PR TITLE
Include multi conf and change address for TB6612FNG

### LINKED ISSUE #2411: Add multiple Grove TB6612FNG drivers
**Describe the problem you have/What new integration you would like**
o Would love to ability to configure multiple instances of the driver boards

**Please describe your use case for this integration and alternatives you've tried:**
 I am making a chemical dispenser for my aquarium and need the ability to drive 4 pumps. To do this I would need two Grove TB6612FNG drivers. Currently, I don't see an option to configure multiples. 

**Additional context**

PXL_20231015_024847871 PORTRAIT
PXL_20231015_024659552 PORTRAIT ORIGINAL
image

### PR DESCRIPTION
**Pull request by max246**
## Description:

Updated the information for the new method and ensure to provide examples how to setup multi boards

**Related issue (if applicable):** fixes  https://github.com/esphome/feature-requests/issues/2411

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#5492

## Checklist:

  - [x] I am merging into `next` because this is new documentation that has a matching pull-request in [esphome](https://github.com/esphome/esphome) as linked above.  
    or
  - [ ] I am merging into `current` because this is a fix, change and/or adjustment in the current documentation and is not for a new component or feature.

  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

**Comment by simonyel** [comment]
I tested the `github://pr#5492,` and it works as expected.

**Review by simonyel — COMMENTED** [review]
I tested the github://pr#5492, and it works as expected. I have two driver boards and 4 DC motors\peri-pumps. I was able to address and execute commands on each device. Start > Break >Stop.

**Review by jesserockz — CHANGES_REQUESTED** [review]
As this is a feature matched with a PR in `esphome/esphome`, please target your PR to the `next` branch and rebase.

Please also update your PR titles to at least mention the component that changes have been made for.

Thanks
Jesse

**Comment by max246** [comment]
> As this is a feature matched with a PR in `esphome/esphome`, please target your PR to the `next` branch and rebase.
> 
> Please also update your PR titles to at least mention the component that changes have been made for.
> 
> Thanks Jesse

thanks! Sorry about that

File: components/grove_tb6612fng.rst [hunk 1]
```diff
@@ -14,11 +14,25 @@ At the current stage of implementation only DC motor is implemented.
...
+    # Example of multi config
+    grove_tb6612fng:
+      - address: 0x14
         id: test_motor
-        name: motor_outside
-        address: 0x14
+
+      - address: 0x15
+        id: test_motor2
+
```
Comment (jesserockz) [inline components/grove_tb6612fng.rst]: This whole example config should be smaller and only include the minimum required for getting the motor working. The rest is up to the user to read the documentation and write their config for their situation.

File: components/grove_tb6612fng.rst [hunk 2]
```diff
@@ -124,6 +137,21 @@ Set the board to be awake, every esphome is restarted the default mode is set to
         - grove_tb6612fng.no_standby
             id: test_motor
 
+.. grove_tb6612fng.change_address:
+
+``grove_tb6612fng.change_address`` Action
+-----------------------------------------
+
+Set the board can change address in case you want to connect multiple once, the addresses avialable are: 0x01 ~ 0x7f
```
Comment (jesserockz) [inline components/grove_tb6612fng.rst]:
```suggestion
If you require connecting multiple boards at once, the address can be changed using this action. The address can be changed to a value in the range of ``0x01 - 0x7f`` inclusive.
```

File: components/grove_tb6612fng.rst [hunk 3]
```diff
@@ -124,6 +137,21 @@ Set the board to be awake, every esphome is restarted the default mode is set to
...
+``grove_tb6612fng.change_address`` Action
+-----------------------------------------
+
+Set the board can change address in case you want to connect multiple once, the addresses avialable are: 0x01 ~ 0x7f 
+
+.. code-block:: yaml
+
+    on_...:
+      then:
+        - grove_tb6612fng.change_address
```
Comment (jesserockz) [inline components/grove_tb6612fng.rst]:
```suggestion
        - grove_tb6612fng.change_address:
```

**Review by jesserockz — CHANGES_REQUESTED** [review]

**Review by jesserockz — APPROVED** [review]

**Pull request merged**